server/models/licenceStages.js

```diff
@@ -10,23 +10,16 @@ const licenceStages = {
 };
 
 const transitions = {
-    CA: {
-        RO: licenceStages.PROCESSING_RO,
-        DM: licenceStages.APPROVAL
+    caToRo: licenceStages.PROCESSING_RO,
+    caToDm: licenceStages.APPROVAL,
+    roToCa: {
+        'default': licenceStages.PROCESSING_CA,
+        addressRejected: licenceStages.ELIGIBILITY,
+        optedOut: licenceStages.ELIGIBILITY
     },
```

Comment:
This has made me think that these two edge cases should have their own transition

Reply:
yes